libtiledbsoma/CMakeLists.txt

```
@@ -173,6 +173,19 @@ message(STATUS "Starting TileDB-SOMA regular build.")
 set(EP_SOURCE_DIR "${EP_BASE}/src")
 set(EP_INSTALL_PREFIX "${EP_BASE}/install")
 
+
+# ###########################################################
+# Check compiler version supports full C++20 standard.
+# ###########################################################
+
+set(GCC_MIN 13.0)
+if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS GCC_MIN)
+        message(FATAL_ERROR "GNU GCC must be at least version ${GCC_MIN}. Using version ${CMAKE_CXX_COMPILER_VERSION}.")
```

```suggestion
        message(FATAL_ERROR "GNU GCC must be at least version ${GCC_MIN}. Found version ${CMAKE_CXX_COMPILER_VERSION}.")
```

"Using" sounds a little like "hey, we're switching over to GCC 13 instead!"

Which is obviously not the case, I grant, since this is a build error. Nonetheless I think "Found" is more clear.